mock-thunderhead/src/main/resources/application.yml

```diff
@@ -27,6 +27,7 @@ auth:
     upgrade.internalrepo.enable: true
     hbase.cloudstorage.enable: true
     datalake.efs.enable: false
+    datalake.customimage.enable: false
```

Comment:
in local mode I guess we should enable this. cc @sodre90

Reply:
Can you confirm should I enable it or not? 

The actual feature is not in committed yet. So this setting does not take effect. Also, this should only affect the mock test, right?

Reply:
we can enable it yes

Reply:
I will do it in https://github.com/hortonworks/cloudbreak/pull/10124 to enable that.